src/background.js

```diff
@@ -3,8 +3,19 @@
  */
 const TRACKING_ID = 'UA-35433268-79';
 
+const KINTO_SERVER = 'https://kinto.dev.mozaws.net/v1';
+// XXX: Read this from Kinto fxa-params
+const FXA_CLIENT_ID = 'c6d74070a481bc10';
+const FXA_OAUTH_SERVER = 'https://oauth-scoped-keys.dev.lcip.org/v1';
```

Comment:
Should these be hardcoded vs pulled/build from some ENV vars?
Seems odd that we'd have a "production-esque" add-on distributed via Test Pilot talking to dev instances of services.

Reply:
The pr won't be merged until this is using prod urls :) it's gonna be using a prod kinto url

Reply:
@vladikoff: OK, cool... but how will that be configured? ENV var? Hard coded?
That means that if we do load testing on this in the future, any load testery would be hitting a prod server endpoint (unless we have flags or we remember to manually edit the endpoint and rebuild add-on locally).

Reply:
Should we put that in the config?